apps/web/components/layout/side-bar.tsx

```
@@ -0,0 +1,175 @@
+            <div className="flex flex-row items-center gap-2 opacity-50">
+              <IconSearch size={16} strokeWidth={2} /> {isSidebarOpen ? 'Search' : ''}
+            </div>
+            {isSidebarOpen && <div className="flex-1" />}
+            {/* {isSidebarOpen && <div className="flex flex-row gap-1">
+              <Kbd className='w-5'><IconCommand className='size-3 shrink-0' /></Kbd>
+              <Kbd>K</Kbd>
+            </div>} */}
+          </Button>
+        </Flex>
+
+        {false ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)